x-pack/plugins/security_solution/public/management/cypress/tasks/isolate.ts

```diff
@@ -139,8 +139,8 @@ export const filterOutIsolatedHosts = (): void => {
   cy.getByTestSubj('querySubmitButton').click();
 };
 
-const checkEndpointListForIsolatedHosts = (expectIsolated: boolean): void => {
-  const chainer = expectIsolated ? 'contain.text' : 'not.contain.text';
```

Comment:
Not really sure why this worked earlier. I actually didn't find anything relating to `contain.text` or `not.contain.text` in cypress docs. Although, it looks like these "chainers" were doing exact string matches instead of substring matches. With the change here we're doing substring matches. So even though the text actually is `statusHealthyIsolated` the assertion would now pass.